sigopt/xgboost/constants.py

```diff
@@ -1,15 +1,18 @@
 DEFAULT_EVALS_NAME = 'TestSet'
 USER_SOURCE_NAME = 'User Specified'
+SIGOPT_DEFAULTS_SOURCE_NAME = 'SigOpt Defaults'
 XGBOOST_DEFAULTS_SOURCE_NAME = 'XGBoost Defaults'
 
 USER_SOURCE_PRIORITY = 2
-XGBOOST_DEFAULTS_SOURCE_PRIORITY = 3
+SIGOPT_DEFAULTS_SOURCE_PRIORITY = 3
+XGBOOST_DEFAULTS_SOURCE_PRIORITY = 4
```

Comment:
Reordering this might screw up the web...cc @simonhowey ?

Reply:
I'll defer to Simon on this. At the very least, it seems to be rendering okay: 
https://app.sigopt.com/run/193376

Reply:
Would this mess up at the Project level?

Reply:
Should be fine the relative number is what matters not the actual number

Reply:
I guess we only need to update the coloring scheme on the web dashboard then.